baselines/ppo2/ppo2.py

```diff
@@ -10,47 +10,113 @@
 from baselines.common.runners import AbstractEnvRunner
 
 class Model(object):
+    """
+    We use this object to :
+    __init__:
+    - Creates the step_model
+    - Creates the train_model
+
+    train():
+    - Make the training part (feedforward and retropropagation of gradients)
+
+    save/load():
+    - Save load the model
+    """
     def __init__(self, *, policy, ob_space, ac_space, nbatch_act, nbatch_train,
                 nsteps, ent_coef, vf_coef, max_grad_norm):
         sess = tf.get_default_session()
 
+        # CREATE OUR TWO MODELS
+        # act_model that is used for sampling
         act_model = policy(sess, ob_space, ac_space, nbatch_act, 1, reuse=False)
+        
+        # Train model for training
         train_model = policy(sess, ob_space, ac_space, nbatch_train, nsteps, reuse=True)
 
+        # CREATE THE PLACEHOLDERS
         A = train_model.pdtype.sample_placeholder([None])
         ADV = tf.placeholder(tf.float32, [None])
         R = tf.placeholder(tf.float32, [None])
+        # Keep track of old actor
         OLDNEGLOGPAC = tf.placeholder(tf.float32, [None])
+        # Keep track of old critic 
         OLDVPRED = tf.placeholder(tf.float32, [None])
         LR = tf.placeholder(tf.float32, [])
+        # Cliprange
         CLIPRANGE = tf.placeholder(tf.float32, [])
 
         neglogpac = train_model.pd.neglogp(A)
+
+        # Calculate the entropy
+        # Entropy is used to improve exploration by limiting the premature convergence to suboptimal policy.
         entropy = tf.reduce_mean(train_model.pd.entropy())
 
+        # CALCULATE THE LOSS
+        # Total loss = Policy gradient loss - entropy * entropy coefficient + Value coefficient * value loss
+       
+        # Clip the value
+        # Get the value predicted
         vpred = train_model.vf
+
+        # Clip the value = Oldvalue + clip(value - oldvalue, min = - cliprange, max = cliprange)
         vpredclipped = OLDVPRED + tf.clip_by_value(train_model.vf - OLDVPRED, - CLIPRANGE, CLIPRANGE)
+        # Unclipped value
         vf_losses1 = tf.square(vpred - R)
+        # Clipped value
         vf_losses2 = tf.square(vpredclipped - R)
+        # Value loss 0.5 * SUM [max(unclipped, clipped)
         vf_loss = .5 * tf.reduce_mean(tf.maximum(vf_losses1, vf_losses2))
+        
+
+        # Remember we want ratio (pi current policy / pi old policy)
+        # But neglopac returns us -log(policy)
```

Comment:
`neglopac` -> `neglogpac`

Reply:
Thanks for the feedback I will correct the typos. Quick question do you think I send a new pull request or I just update these ?

Reply:
I've never dealt with this repo before, but usually I'd just update the existing PR.